distro/adaptation/fedora

```diff
@@ -56,6 +56,7 @@ libfabric-dev: libfabric-devel
 libfdt-dev: libfdt-devel
 libfuse-dev: fuse-devel
 libgbm-dev: mesa-libgbm-devel
+libgmp-dev: gmp-devel
 libgl1-mesa-dev: mesa-libGL-devel
```

Comment:
for the patch subject, can  you name it like

distro/adaptation/fedora: libgmp-dev package adaptation

Reply:
Done.